src/async_std.rs

```diff
@@ -146,15 +146,21 @@ pub(crate) mod dummy_tls {
 }
 
 #[cfg(not(any(feature = "async-tls", feature = "async-native-tls")))]
-use self::dummy_tls::{client_async_tls_with_connector_and_config, AutoStream};
+pub use self::dummy_tls::client_async_tls_with_connector_and_config;
```

Comment:
Can you do the same in the tokio module, and also make sure that this re-exported function is documented in all configurations?

Reply:
If only the dummy one is enabled then it won't have documentation. You'll have to copy the docs to every variant of the function :)

Reply:
Aha, I see.

Reply:
Done. Confirmed by

```
git grep -B 'fn client_async_tls_with_connector_and_config'
```

```
src/async_std.rs-    /// Creates a WebSocket handshake from a request and a stream,
src/async_std.rs-    /// upgrading the stream to TLS if required and using the given
src/async_std.rs-    /// connector and WebSocket configuration.
src/async_std.rs:    pub async fn client_async_tls_with_connector_and_config<R, S>(
--
src/async_std.rs-    /// Creates a WebSocket handshake from a request and a stream,
src/async_std.rs-    /// upgrading the stream to TLS if required and using the given
src/async_std.rs-    /// connector and WebSocket configuration.
src/async_std.rs:    pub(crate) async fn client_async_tls_with_connector_and_config<R, S>(
--
src/async_tls.rs-/// Creates a WebSocket handshake from a request and a stream,
src/async_tls.rs-/// upgrading the stream to TLS if required and using the given
src/async_tls.rs-/// connector and WebSocket configuration.
src/async_tls.rs:pub async fn client_async_tls_with_connector_and_config<R, S>(
--
src/tokio/async_tls.rs-/// Creates a WebSocket handshake from a request and a stream,
src/tokio/async_tls.rs-/// upgrading the stream to TLS if required and using the given
src/tokio/async_tls.rs-/// connector and WebSocket configuration.
src/tokio/async_tls.rs:pub async fn client_async_tls_with_connector_and_config<R, S>(
--
src/tokio/dummy_tls.rs-/// Creates a WebSocket handshake from a request and a stream,
src/tokio/dummy_tls.rs-/// upgrading the stream to TLS if required and using the given
src/tokio/dummy_tls.rs-/// connector and WebSocket configuration.
src/tokio/dummy_tls.rs:pub(crate) async fn client_async_tls_with_connector_and_config<R, S>(
--
src/tokio/native_tls.rs-/// Creates a WebSocket handshake from a request and a stream,
src/tokio/native_tls.rs-/// upgrading the stream to TLS if required and using the given
src/tokio/native_tls.rs-/// connector and WebSocket configuration.
src/tokio/native_tls.rs:pub async fn client_async_tls_with_connector_and_config<R, S>(
--
src/tokio/openssl.rs-/// Creates a WebSocket handshake from a request and a stream,
src/tokio/openssl.rs-/// upgrading the stream to TLS if required and using the given
src/tokio/openssl.rs-/// connector and WebSocket configuration.
src/tokio/openssl.rs:pub async fn client_async_tls_with_connector_and_config<R, S>(
--
src/tokio/rustls.rs-/// Creates a WebSocket handshake from a request and a stream,
src/tokio/rustls.rs-/// upgrading the stream to TLS if required and using the given
src/tokio/rustls.rs-/// connector and WebSocket configuration.
src/tokio/rustls.rs:pub async fn client_async_tls_with_connector_and_config<R, S>(
```